src/components/user/tour/tour.js

```diff
@@ -1,6 +1,67 @@
 import React from "react";
-import Typography from "@material-ui/core/Typography";
+import { DataGrid } from '@material-ui/data-grid';
 
-export default function Tour() {
-  return <Typography paragraph>The 'Tour' Page comming soon...</Typography>;
+const rows = [
+  {
+    id: 1, 
+    status: 'All booked',
+    name: 'Bellinis in Armenia',
+    origin: 'Italy',
+    days: '7',
+    date: 'Jan 28-Feb 11',
+    destination: 'Yerevan, Sevan, Dilijan',
+    guests: 'Marco Belini, Chiara Belini',
+    hotels: 'Noy Land, Popock 1',
+    activities: 'Yell Park, Papanino',
+    flight: 'QR 706',
+    price: '$5400',
+    notes: 'Very picky person',
+    createdBy: 'Armine G',
+   },
+   {
+    id: 1, 
```

Comment:
You use `id` property as a react key, please do not repeat it. Keys must be unique values, in your case you have `id: 1` twice